lib/internal/module.js

```diff
@@ -44,3 +48,27 @@ function stripBOM(content) {
   }
   return content;
 }
+
+exports.builtinLibs = ['assert', 'buffer', 'child_process', 'cluster',
+  'crypto', 'dgram', 'dns', 'domain', 'events', 'fs', 'http', 'https', 'net',
+  'os', 'path', 'punycode', 'querystring', 'readline', 'stream',
+  'string_decoder', 'tls', 'tty', 'url', 'util', 'v8', 'vm', 'zlib'];
```

Comment:
Another good excuse for https://github.com/nodejs/node/issues/3307?

I guess it doesn't matter too much. The only thing that is remotely on the roadmap as a module is `async_wrap`.